include/wlr/types/wlr_output.h

```diff
@@ -107,6 +107,7 @@ struct wlr_output {
 
 	struct wl_list cursors; // wlr_output_cursor::link
 	struct wlr_output_cursor *hardware_cursor;
+	int hardware_cursors_enabled; // > 0 when enabled
```

Comment:
I think this variable name is a little misleading. If it were just "are hardware cursors enabled" it should be a bool. I'd rather be more transparent about how it's used.

Reply:
Since it's roughly capturing `enable requests - disable requests`, maybe a better name could be `hardware_cursors_tokens` or `hardware_cursors_enable_requests`?

Would it make sense for the value to be > 1? If not, why not enable them at 0, and count disable requests?